### PR TITLE
iotaized key prefixes

### PR DESCRIPTION
**Pull request by mvid**

### CODE REVIEW

File: module/x/gravity/types/key.go [hunk 1]
```diff
@@ -174,10 +135,10 @@ func GetClaimKey(details EthereumClaim) []byte {
 // validator X and validator y where making different claims about the same event nonce
 // Note that the claim hash does NOT include the claimer address and only identifies an event
 func GetAttestationKey(eventNonce uint64, claimHash []byte) []byte {
-	key := make([]byte, len(OracleAttestationKey)+len(UInt64Bytes(0))+len(claimHash))
-	copy(key[0:], OracleAttestationKey)
-	copy(key[len(OracleAttestationKey):], UInt64Bytes(eventNonce))
-	copy(key[len(OracleAttestationKey)+len(UInt64Bytes(0)):], claimHash)
+	key := make([]byte, 1 + len(UInt64Bytes(0)) + len(claimHash))
```
Comment (jackzampolin) [inline module/x/gravity/types/key.go:138]: isn't `sdk.Uint64ToBigEndian` 8 bytes long?

**Review by jackzampolin — APPROVED** [review]
A good first step

File: module/x/gravity/types/key.go (hunk 1 above)
Comment (mvid) [reply module/x/gravity/types/key.go:138]: All of the existing key prefixes were one byte long, so len(KeyPrefix) would always be 1. That is all I replaced here.

**Review by jackzampolin — APPROVED** [review]

**Comment by jackzampolin** [comment]
Going to go ahead and merge this into my branch.

**Pull request merged**